profile/README.md

```diff
@@ -6,31 +6,22 @@
 We're a collection of open source tools that create, configure, and work with containers.
 
 ## 🛠️ Tools
-Some of our tools include [Podman](https://github.com/containers/podman), [Buildah](https://github.com/containers/buildah), [Skopeo](https://github.com/containers/skopeo), [conmon-rs](https://github.com/containers/conmon-rs), [crun](https://github.com/containers/crun), [Podman Desktop](https://github.com/containers/podman-desktop), [youki](https://github.com/containers/youki) and [RamaLama](https://github.com/containers/ramalama) as well as our supporting libraries common, storage, image in the [container-libs](https://github.com/containers/container-libs) repo. Check them out and see all the cool things happening!
+Some of our tools include [Podman](https://github.com/containers/podman), [Buildah](https://github.com/containers/buildah), [Skopeo](https://github.com/containers/skopeo), [conmon-rs](https://github.com/containers/conmon-rs), [crun](https://github.com/containers/crun), and [RamaLama](https://github.com/containers/ramalama) as well as our supporting libraries common, storage, image in the [container-libs](https://github.com/containers/container-libs) repo. Check them out and see all the cool things happening!
```

Comment:
I'm fine with this change, but I think we should have an addition.  Something along the lines of "These projects used to live in this organization, and now they can be found at CNCF" or some such, and then include appropriate links.

Or better yet, have a new section just below this and move the logos there.

I come to this page all the time to get to the different projects, including Podman and Buildah.  I'd be unhappy if the links were gone without some form of breach crumbs leading me to the new location.